scripts/dts/extract/globals.py

```diff
@@ -333,6 +333,11 @@ def build_cell_array(prop_array):
     index = 0
     ret_array = []
 
+    if isinstance(prop_array, int):
+    # Work around old code generating an integer for e.g.
+    # 'pwms = <&foo>'
```

Comment:
Wrong indentation for comment. Looks fine otherwise.

Thanks!

Reply:
Could you squash the indentation fix so it's just a single commit too?

Can `git push --force` after that.

Reply:
Could make the commit title something like

    scripts: dts: Fix generation of single-value phandle/value array

to make CI happy.